Tests/UI/New UI/BillingForm/View/ExpiryDateViewTests.swift

```diff
@@ -87,5 +89,4 @@ class ExpiryDateViewTests: XCTestCase {
         view.updateExpiryDate(to: "Test")
         XCTAssertFalse(view.style?.error?.isHidden ?? true)
     }
-
-}
+}
```

Comment:
not a blocker, but we should have newlines at the end of our files

Reply:
```suggestion
}

```